Makefile.am

```diff
@@ -40,6 +40,7 @@ EXTRA_DIST = \
 clean-local:
 	find $(builddir) -name '*.gc??' -delete
 	find $(srcdir) -name '*.pyc' -delete
+	find $(builddir)/dist -name README.md -prune -o -name dist -prune -o -depth -delete
```

Comment:
How can files named `README.md` and `dist` end up in dist? Can't we just delete the whole directory unconditionally?